unity-client/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatMessage/ChatMessage.cs

```diff
@@ -11,6 +11,14 @@ public enum Type
             SYSTEM
         }
 
+        public ChatMessage() { }
```

Comment:
Why is it necessary to create an empty constructor? Is it only for standard conventions? or is it being used in some place?

Reply:
So we can use `object initializers` like ` new ChatMessage { messageType = " type" } ;